### ISSUE TITLE
Unable to test the bot

### ISSUE THREAD

**Issue by ramesh8830** [theopenconversationkit/tock-docker#32]
Hi

Unable to test the bot from the demo url provided on the repo and tried installing docker also giving me the same error

![image](https://github.com/theopenconversationkit/tock-docker/assets/56535842/8feb0861-1c65-4f56-8985-c2d9f7631b23)

![image](https://github.com/theopenconversationkit/tock-docker/assets/56535842/ff9c1be9-2222-49ab-bd7d-ca61560c48c1)

```
admin_web_1        | 2023-10-14T20:24:02.569 [vert.x-worker-thread-5] INFO  ai.tock.shared.Retrofits - {"userId":"test_652aeff8dcad016a741a73a1_en__h9q2pyygh","recipientId":"test_bot_652aeff8dcad016a741a73a1_en","message":{"text":"Hi","messages":[],"eventType":"sentence","delay":0},"targetConnectorType":{"id":"web","userInterfaceType":"textChat"},"test":false,"debugEnabled":false}
admin_web_1        | 2023-10-14T20:24:02.569 [vert.x-worker-thread-5] INFO  ai.tock.shared.Retrofits - --> END POST (280-byte body)
admin_web_1        | 2023-10-14T20:24:02.572 [vert.x-eventloop-thread-0] INFO  ai.tock.bot.admin.BotAdminVerticle - Error 405: /io/andi/test/test-andi/en
admin_web_1        | 2023-10-14T20:24:02.573 [vert.x-worker-thread-5] INFO  ai.tock.shared.Retrofits - <-- 405 Method Not Allowed http://localhost:8080/io/andi/test/test-andi/en (3ms)
admin_web_1        | 2023-10-14T20:24:02.573 [vert.x-worker-thread-5] INFO  ai.tock.shared.Retrofits - content-type: text/plain
admin_web_1        | 2023-10-14T20:24:02.573 [vert.x-worker-thread-5] INFO  ai.tock.shared.Retrofits -
admin_web_1        | 2023-10-14T20:24:02.573 [vert.x-worker-thread-5] INFO  ai.tock.shared.Retrofits - Error 405: Method Not Allowed
admin_web_1        | 2023-10-14T20:24:02.573 [vert.x-worker-thread-5] INFO  ai.tock.shared.Retrofits - <-- END HTTP (29-byte body)
admin_web_1        | 2023-10-14T20:24:02.574 [vert.x-worker-thread-5] ERROR ai.tock.bot.admin.test.TestCoreService - error with BotApplicationConfiguration(applicationId=test-AndI, botId=andi, namespace=AndI, nlpModel=andi, connectorType=rest, ownerConnectorType=web, name=AndI, baseUrl=http://localhost:8080, parameters={X-Auth-Token=}, path=/io/andi/test/test-andi, _id=652aeff8dcad016a741a73a1, targetConfigurationId=652aeff8dcad016a741a73a0) : Error 405: Method Not Allowed
```

by the way, how to change the default username and password?

**Comment by FBergeron** [comment]
I think I'm having the same issue.  

I start Tock from docker doing:

docker-compose up

The web interface works fine.  I can add intents normally.  However, when I try to execute a test, I get a Technical Error :( ] exactly as depicted in the previous message.  I also have a 405 error in the console as well.

I'm using MacOS on M1 processor.  I wonder if it could cause some issue?

**Comment by vsct-jburet** [comment]
For M1/M2 you need to pass a special env var:

`PLATFORM=arm64v8/ docker compose -f docker-compose-bot-open-data.yml up`

I have fixed a kotlin lib version issue in tock 23.9.1 (the tock/bot_open_data fails at startup). Please test again with 23.9.1 and reopen the issue if the bug is still there

![image](https://github.com/theopenconversationkit/tock-docker/assets/12413454/781fb67f-b1ee-4fd1-a6b3-29d83497c5b7)

**Comment by FBergeron** [comment]
Just to confirm that it's working now.  Thank you!

**Comment by mvarchdev** [comment]
For me it is not working, tried everything what is described also here and here: https://github.com/theopenconversationkit/tock/issues/1368

I have Macbook pro M2 max

**Comment by mvarchdev** [comment]
@vsct-jburet could you please help me? Thanks in advance!

**Comment by vsct-jburet** [comment]
I open a new issue https://github.com/theopenconversationkit/tock-docker/issues/35

**Comment by vsct-jburet** [comment]
please add your docker logs on the new ticket ;)

**Comment by shadow3696** [comment]
> For M1/M2 you need to pass a special env var:
> 
> `PLATFORM=arm64v8/ docker compose -f docker-compose-bot-open-data.yml up`
> 
> I have fixed a kotlin lib version issue in tock 23.9.1 (the tock/bot_open_data fails at startup). Please test again with 23.9.1 and reopen the issue if the bug is still there
> 
> ![image](https://private-user-images.githubusercontent.com/12413454/285540326-781fb67f-b1ee-4fd1-a6b3-29d83497c5b7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ2Mzk0MDgsIm5iZiI6MTczNDYzOTEwOCwicGF0aCI6Ii8xMjQxMzQ1NC8yODU1NDAzMjYtNzgxZmI2N2YtYjFlZS00ZmQxLWE2YjMtMjlkODM0OTdjNWI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE5VDIwMTE0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFiYmQ4MjUzMjgzYzViOWUwYjMzYjFhMDE3YTBkYWM5M2MwZjNkNWQyMWMxMmQzYzliMDg3YjUwMGQ2ZmJiODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uRdQU0B7F5x7iC3Q-tdgIgvDJLyrXttZu9qsOpuH2-o)

i have same error but im not working on mac im on ubuntu and i have to do the same thing or i have to do something diffrent
